infra/modules/au_fhir_inferno/manifests/inferno/postgres.stateful.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: StatefulSet ## allows for stable data storage that persists across pod restarts and stable network for later clusterd Postgres deployments
+metadata:
+  name: inferno-postgres
+  namespace: ${namespace}
+spec:
+  serviceName: "inferno-postgres"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: inferno-postgres
+  template:
+    metadata:
+      labels:
+        app: inferno-postgres
+    spec:
+      nodeSelector:
+        kubernetes.io/arch: amd64
+      containers:
+        - name: postgres
+          image: postgres:16.2-bullseye
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 5432
+          envFrom:
+            - secretRef:
+                name: postgres-database
+          volumeMounts:
+            - mountPath: "/data" # not sure why this isn't the default "/var/lib/postgresql/data" ??
```

Comment:
@KyleOps I think it is a typo, It definatly should be /var/lib/postgresql/data.
Could you please update.